nemo/core/classes/modelPT.py

```diff
@@ -1019,3 +1226,13 @@ def __is_model_being_restored() -> bool:
     def __set_model_restore_state(is_being_restored: bool):
         global _MODEL_IS_RESTORED
         _MODEL_IS_RESTORED = is_being_restored
+
+    @staticmethod
+    def __set_eff_save(use_eff_save: bool):
```

Comment:
Do not use __ here, it makes them name mangled and users cannot use them from ModelPT or it's subclasses.

Reply:
wasn't sure how/where you want to use it, but ok, you are right, let's make it "public"

Reply:
done